eng/tools/versioning/set-version.js

```diff
@@ -72,8 +72,8 @@ async function main(argv) {
     artifactName,
     repoRoot,
     newVersion,
-    false,
-    true,
+    "-Unreleased:$false",
+    "-ReplaceLatestEntryTitle:$true",
```

Comment:
Part of the reason you use switches so you don't have to pass the value. If it isn't pass then it is false but if it is passed then it is true.

Reply:
Yes but the again the way we are calling it in various places requires that something always be passed. So we cant just leave it out.

Reply:
I changed the parameters to Boolean.